src/Processors/Executors/ExecutionThreadContext.cpp

```diff
@@ -75,10 +79,17 @@ bool ExecutionThreadContext::executeTask()
     }
 
     if (profile_processors)
-        node->processor->elapsed_us += execution_time_watch->elapsedMicroseconds();
+    {
+        UInt64 elapsed_microseconds =  execution_time_watch->elapsedMicroseconds();
+        node->processor->elapsed_us += elapsed_microseconds;
+        span.addAttribute("execution_time_ns", elapsed_microseconds);
```

Comment:
Why execution_time_**ns**, but elapsed_**microseconds** ?
Also if `bool(profile_processors) == true` and `NDEBUG` attributes would be set twice, it it ok?

Reply:
Will fix.